python/setup.py

```diff
@@ -96,7 +96,7 @@ def config_cython():
                 library_dirs=library_dirs,
                 libraries=libraries,
                 language="c++"))
-        return cythonize(ret)
+        return cythonize(ret, compiler_directives={"language_level": 3})
```

Comment:
Sorry I am not a cython expert. Is this something like opt level?

Reply:
It directs it to use Python 3 silencing a warning/issue.